content-enricher/content-enricher.yml

```diff
@@ -0,0 +1,6 @@
+tagline: Add data from message
```

Comment:
Should this be `Add data to message` ?

Reply:
Could be better if `Add extra data to a message`

Reply:
+1